tests/test_params2csv.py

```diff
@@ -165,28 +174,124 @@ def test_ert_workflow(tmp_path):
             f"real\t{i}", encoding="utf8"
         )
 
-    Path("PARAMS2CSV_ITER0").write_text(
+    Path("PARAMS2CSV").write_text(
```

Comment:
I think you might actually want to keep this different from `PARAMS2CSV` since that is the forward model name. Maybe it doesn't make a difference... `PARAMS2CSV_ITER0` is what is suggested in the documentation text too.

Reply:
I think my reasoning was that most users probably want to extract one csv-file for all iterations, and not one per iteration.  At least now after standardizing the csv-file and adding more case metadata in the output. But the tests are not where most users go to see how to use this tool anyway.. 😄 
Maybe we should update the ERT WORKFLOW documentation to show how to extract all iterations in one go to the users instead? As the documentation is today, many would probably think that they would have to have multiple ERT WORKFLOWS to extract all the iterations. Agree @alifbe @rnyb ?

Reply:
I don't have a strong opinion on this, but can it be potentially be a problem if for instance mixing of hist and pred?

Reply:
Anyways, I'll guess the user can still choose however they want to use it